tests/ssh/test_ssh_default_password.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import paramiko
+import logging
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.topology("any")
+]
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_USERNAME = "admin"
+DEFAULT_PASSWORD_PUBLIC = "YourPaSsWoRd"
```

Comment:
Can we make this general?
The default password is a build configuration option, which means everyone can change it. In public, it is YourPaSsWoRd; In MS, it is password; but how about other orgs or companies?
Is possiable to have a configure file(testbed.yml?) and then we can let user choice them own favors and we can also avoid expose internal descison here.

Reply:
Yeah, thanks for your suggestion. 
I will just put the public info in the public repo and provide a general method to get the default password according to the image version(In public, only get `YourPaSsWoRd`). And after merging into the internal repo, I will append the internal info. So that it can avoid exposing internal decisions. It also applied to other vendors.
For testbed.yml, it is not appreciated to put the SSH password, because it only set some static info but the image version may be changed.